examples/large_models/inferentia2/llama2/continuous_batching/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-inference-neuronx:1.13.1-neuronx-py310-sdk2.15.0-ubuntu20.04
```

Comment:
I guess this won't work with sdk 2.15; this is first introduced in neuron sdk 2.16 release.